assets/data/cards_pt-br.json

```
@@ -202,7 +202,7 @@
     {
       "title": "React Native",
       "description": "É um framework do Javascript baseado no React para desenvolvimento de aplicativos móveis multiplataforma.",
-      "tags": ["Conceito", "Front-end"]
+      "tags": ["Conceito", "Mobile"]
```

Analisando as tags de outros frameworks já presentes no Diciotech, que constam no `cards_pt-br.json`, eles não possuem a tag "Conceito". Nesse caso, pode deixar apenas a tag "Mobile".

```suggestion
      "tags": ["Mobile"]
```